CHANGELOG.md

```diff
@@ -2,6 +2,27 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+### Mapbox Navigation SDK 1.2.0-alpha.3 - October 21, 2020
+
+Visit our [1.0.0 migration guide](https://github.com/mapbox/mapbox-navigation-android/wiki/1.0-Navigation-SDK-Migration-Guide) and the [documentation pages](http://docs.mapbox.com/android/navigation/overview) for more information.
+
+### Changelog
+
+#### Features
+* Update the algorithm to calculate the vanishing point of the route to use EPSG:3857 projection instead of turf measurements. The precision is now spot on, even on extremely long routes, it even greatly outperforms the precision of the over zoomed map tiles. It also improves the performance of the `MapRouteLine` [#3661](https://github.com/mapbox/mapbox-navigation-android/pull/3661)
```

Comment:
```suggestion
* Update the algorithm to calculate the vanishing point of the route to use EPSG:3857 projection instead of turf measurements. This greatly increases the precision, even on extremely long routes. It also improves the performance of the `MapRouteLine`. [#3661](https://github.com/mapbox/mapbox-navigation-android/pull/3661)
```

Also, I'd title it _Improvements_, not _Features_.

Reply:
done